src/pages/profile.js

```diff
@@ -38,6 +39,7 @@ const ProfilePage = () => {
       <Router basepath="/profile">
         <ProfileIndexPage path="/" />
         <ProfileEditEmailPage path="/edit" />
```

Comment:
This path should be refactored to `/email`.

Reply:
👍